railties/lib/rails/generators/actions.rb

```diff
@@ -215,7 +215,7 @@ def generate(what, *args)
         log :generate, what
         argument = args.flat_map(&:to_s).join(" ")
 
-        in_root { run_ruby_script("bin/rails generate #{what} #{argument}", verbose: false) }
+        in_root { run_ruby_script("rails generate #{what} #{argument}", verbose: false) }
```

Comment:
My above thinking would apply here too... though I note the below `rails_command` seems happy relying on the unqualified name 🤠